modules/ossm-config-kiali-otel.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/kiali/ossm-kiali-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-config-kiali-otel_{context}"]
+= Configuring Kiali with {DTProductName}
+
+When Kiali is properly configured with {DTProductName}, you can view distributed traces in the Kiali console. These traces provide insight into the communication between services within the service mesh, helping you understand how requests are flowing through your system and where potential issues may lie.
+
+The {DTShortName} integration allows Kiali to display trace overlays and details on the graph, scatterplot charts and in-depth trace/span information on detail pages, and integrated span information in logs and metric charts. Kiali also offers links to the external tracing UI.
```

Comment:
```suggestion
```
Too much concept before the procedure. I would move this to another module, or plop it at the end somewhere?